src/components/Dashboard/widgets/widgetDefinitions.js

```diff
@@ -161,6 +158,20 @@ export const WIDGET_DEFINITIONS = [
       }
     ]
   },
+  {
+    type: 'SET_COMMAND',
```

Comment:
This is just me being picky and I understand it's due to the set attribute button being duplicated for this ticket but I think it should be execute command rather than set. Set attribute makes sense because it's setting the value, setting a command isn't really a thing.